<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AskForLODDialog</class>
 <widget class="QDialog" name="AskForLODDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>330</width>
    <height>269</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LOD file processing</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QCheckBox" name="useLODCheck">
     <property name="toolTip">
      <string>Use existing Level Of Distance files</string>
     </property>
     <property name="text">
      <string>Use existing LOD files</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="buildLODCheck">
     <property name="toolTip">
      <string>Build and use LOD file</string>
     </property>
     <property name="text">
      <string>Build and use LOD files</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="saveCompLODCheck">
     <property name="toolTip">
      <string>Save one file containing all LOD levels</string>
     </property>
     <property name="text">
      <string>Save compound LOD file</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="cancelButton">
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="okButton">
     <property name="text">
      <string>OK</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
